README.md

```diff
@@ -1,6 +1,9 @@
 ## PyTorch Docker image
 
-[![Docker Automated build](https://img.shields.io/docker/automated/anibali/pytorch.svg)](https://hub.docker.com/r/anibali/pytorch/)
+[![Docker Automated build](https://img.shields.io/docker/automated/anibali/pytorch)](https://hub.docker.com/r/anibali/pytorch/)
+[![Docker Automated build](https://img.shields.io/docker/image-size/anibali/pytorch)](https://hub.docker.com/r/anibali/pytorch/)
```

Comment:
This also needs to be changed: `https://img.shields.io/docker/image-size/anibali/pytorch/latest`